app/schema.graphql

```diff
@@ -534,6 +534,27 @@ type Point3D {
   z: Float!
 }
 
+type Project implements Node {
+  id: GlobalID!
+  name: String!
+  startTime: DateTime
+  endTime: DateTime
```

Comment:
i would recommend `stop_time` over `end_time` for clarity, but I understand that not everyone has the same preference.

Advantages
- `stop_time` is easy to compute (see `ceil` below)
- `stop_time` implies a convention of right-open interval

```python-console
>>> pd.Timestamp.now(timezone.utc).floor('1d').to_pydatetime()
datetime.datetime(2024, 3, 5, 0, 0, tzinfo=datetime.timezone.utc)
>>> pd.Timestamp.now(timezone.utc).ceil('1d').to_pydatetime()
datetime.datetime(2024, 3, 6, 0, 0, tzinfo=datetime.timezone.utc)
```